plugins/extensions/plugins/ZeroEQPlugin.cpp

```diff
@@ -262,10 +274,58 @@ void ZeroEQPlugin::_transferFunction1DUpdated( )
     transferFunction.resample();
 
     scene->commitSimulationData();
+    scene->commitTransferFunctionData();
     _extensionParameters.engine->getRenderer()->commit();
     _extensionParameters.engine->getFrameBuffer()->clear();
 }
 
+void ZeroEQPlugin::_LookupTable1DUpdated( )
+{
+    ScenePtr scene = _extensionParameters.engine->getScene();
+    TransferFunction& transferFunction = scene->getTransferFunction();
+
+    transferFunction.clear();
+    Vector4fs& diffuseColors = transferFunction.getDiffuseColors();
+    floats& emissionIntensities = transferFunction.getEmissionIntensities();
+
+    const uint8_ts& lut = _remoteLookupTable1D.getLutVector();
+    for( size_t i = 0; i < lut.size(); i += 4 )
+    {
+        Vector4f color = {
+            lut[i*4 + 0 ] / 255.f,
```

Comment:
You're absolutely right!! (Feeling ashamed). That explains why the transfer function was looking weird! Thanks for that!